### ISSUE TITLE
Cannot Login 

### ISSUE THREAD

**Issue by ZhiyuXu0124** [exadel-inc/CompreFace#747]
**Describe the bug**
Unable to log in on the UI interface, it was normal before. But the core functions are normal, I still can call the previous api-key through python-sdk.

**To Reproduce**
1. Go to http://localhost:8000/login
2. Click on Lonin
3. See error

**Error log**
```shell
172.18.0.1 - CommonClientId [10/Mar/2022:07:39:21 +0000] "POST /admin/oauth/token HTTP/1.1" 502 552 "http://localhost:8000/login" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.51 Safari/537.36"

2022/03/10 07:39:21 [error] 22#22: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: ui, request: "POST /admin/oauth/token HTTP/1.1", upstream: "http://172.18.0.5:8080/oauth/token", host: "localhost:8000", referrer: "http://localhost:8000/login"
```

**Screenshots**
![Login](https://user-images.githubusercontent.com/56537567/157611992-64155357-b03c-48ff-bb6d-d67315c5be42.png)
![docker](https://user-images.githubusercontent.com/56537567/157612417-3c3c1efe-d1e3-4f4f-b90b-cad8c9ceb4b1.png)

**Desktop (please complete the following information):**
 - OS: Win10
 - Browser chrome

**Comment by pospielov** [comment]
Could you send the result of `docker logs compreface-admin` command? Or just show `compreface-admin` logs,

**Comment by ZhiyuXu0124** [comment]
```shell
Listening for transport dt_socket at address: 5005
2022-03-11 06:48:50.249  INFO 8 --- [kground-preinit] o.h.validator.internal.util.Version      : HV000001: Hibernate Validator 6.1.5.Final

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::        (v2.3.4.RELEASE)

2022-03-11 06:48:51.414  INFO 8 --- [           main] com.exadel.frs.FrsApplication            : The following profiles are active: dev
2022-03-11 06:48:54.258  INFO 8 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFERRED mode.
2022-03-11 06:48:54.590  INFO 8 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 310ms. Found 10 JPA repository interfaces.
2022-03-11 06:48:55.290  INFO 8 --- [           main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=e7f41a96-3f70-3efd-b7dc-40aa4699bf29
2022-03-11 06:48:55.607  INFO 8 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'com.exadel.frs.commonservice.system.feign.ApperyStatisticsClient' of type [org.springframework.cloud.openfeign.FeignClientFactoryBean] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2022-03-11 06:48:56.658  INFO 8 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2022-03-11 06:48:56.677  INFO 8 --- [           main] o.a.coyote.http11.Http11NioProtocol      : Initializing ProtocolHandler ["http-nio-8080"]
2022-03-11 06:48:56.678  INFO 8 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2022-03-11 06:48:56.678  INFO 8 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.38]
2022-03-11 06:48:56.766  INFO 8 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2022-03-11 06:48:56.766  INFO 8 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 5290 ms
2022-03-11 06:48:57.380  INFO 8 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2022-03-11 06:48:57.652  INFO 8 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2022-03-11 06:48:58.759  INFO 8 --- [           main] liquibase.lockservice                    : Waiting for changelog lock....
2022-03-11 06:49:08.762  INFO 8 --- [           main] liquibase.lockservice                    : Waiting for changelog lock....
2022-03-11 06:49:18.764  INFO 8 --- [           main] liquibase.lockservice                    : Waiting for changelog lock....
2022-03-11 06:49:28.767  INFO 8 --- [           main] liquibase.lockservice                    : Waiting for changelog lock....
2022-03-11 06:49:38.769  INFO 8 --- [           main] liquibase.lockservice                    : Waiting for changelog lock....
2022-03-11 06:49:48.774  INFO 8 --- [           main] liquibase.lockservice                    : Waiting for changelog lock....
2022-03-11 06:49:58.781  INFO 8 --- [           main] liquibase.lockservice                    : Waiting for changelog lock....
2022-03-11 06:50:08.791  INFO 8 --- [           main] liquibase.lockservice                    : Waiting for changelog lock....
2022-03-11 06:50:18.794  INFO 8 --- [           main] liquibase.lockservice                    : Waiting for changelog lock....
2022-03-11 06:50:28.796  INFO 8 --- [           main] liquibase.lockservice                    : Waiting for changelog lock....
2022-03-11 06:50:38.799  INFO 8 --- [           main] liquibase.lockservice                    : Waiting for changelog lock....
2022-03-11 06:50:48.801  INFO 8 --- [           main] liquibase.lockservice                    : Waiting for changelog lock....
2022-03-11 06:50:58.804  INFO 8 --- [           main] liquibase.lockservice                    : Waiting for changelog lock....
```

> Could you send the result of `docker logs compreface-admin` command? Or just show `compreface-admin` logs,

**Comment by pospielov** [comment]
Looks like the container was stopped during the migration process. Did you try to update CompreFace?
This is bad, I hope the DB is not corrupted. 
The easiest way is just to stop CompreFace, delete DB and start it again. But I suppose you don't want to delete the data.
So you need to unblock DB manually.
0. Stop CompreFace
1. You need to update your docker-compose.yml file. Add:
``` 
ports:
      - "6432:5432"
```
to `compreface-postgres-db` container, like here:
https://github.com/exadel-inc/CompreFace/blob/master/dev/docker-compose.yml

2. Connect to DB:
user/pass: postgres/postgres
port: 6432
db: frs

3. Then find table `public.databasechangeloglock` and delete the row

4. Then you can start CompreFace again

**Comment by Omar-Belghaouti** [comment]
@ZhiyuXu0124 @pospielov I did the same steps and I still get the same error, here's what I get after running `docker logs compreface-admin`:
```sh
Listening for transport dt_socket at address: 5005
2022-03-27 18:08:09.025  INFO 8 --- [kground-preinit] o.h.validator.internal.util.Version      : HV000001: Hibernate Validator 6.1.5.Final

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::        (v2.3.4.RELEASE)

2022-03-27 18:08:12.454  INFO 8 --- [           main] com.exadel.frs.FrsApplication            : The following profiles are active: dev
2022-03-27 18:08:22.609  INFO 8 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFERRED mode.
2022-03-27 18:08:24.125  INFO 8 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 1454ms. Found 10 JPA repository interfaces.
2022-03-27 18:08:25.812  INFO 8 --- [           main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=e7f41a96-3f70-3efd-b7dc-40aa4699bf29
2022-03-27 18:08:26.535  INFO 8 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'com.exadel.frs.commonservice.system.feign.ApperyStatisticsClient' of type [org.springframework.cloud.openfeign.FeignClientFactoryBean] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2022-03-27 18:08:29.788  INFO 8 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2022-03-27 18:08:29.840  INFO 8 --- [           main] o.a.coyote.http11.Http11NioProtocol      : Initializing ProtocolHandler ["http-nio-8080"]
2022-03-27 18:08:29.842  INFO 8 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2022-03-27 18:08:29.843  INFO 8 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.38]
2022-03-27 18:08:30.658  INFO 8 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2022-03-27 18:08:30.664  INFO 8 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 18040 ms
2022-03-27 18:08:32.919  INFO 8 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2022-03-27 18:08:34.366 ERROR 8 --- [           main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Exception during pool initialization.

org.postgresql.util.PSQLException: Connection to compreface-postgres-db:6432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:303)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:51)
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:225)
	at org.postgresql.Driver.makeConnection(Driver.java:465)
	at org.postgresql.Driver.connect(Driver.java:264)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:358)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:477)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:560)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:112)
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:272)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1853)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1790)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:594)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:516)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:324)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:322)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:311)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1109)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:869)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:551)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:143)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:758)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:750)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1237)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at com.exadel.frs.FrsApplication.main(FrsApplication.java:28)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:107)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:88)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at java.base/java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.base/java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.base/java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.base/java.net.SocksSocketImpl.connect(Unknown Source)
	at java.base/java.net.Socket.connect(Unknown Source)
	at org.postgresql.core.PGStream.createSocket(PGStream.java:231)
	at org.postgresql.core.PGStream.<init>(PGStream.java:95)
	at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:98)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:213)
	... 41 common frames omitted

2022-03-27 18:08:34.372  WARN 8 --- [           main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in class path resource [org/springframework/boot/autoconfigure/liquibase/LiquibaseAutoConfiguration$LiquibaseConfiguration.class]: Invocation of init method failed; nested exception is liquibase.exception.DatabaseException: org.postgresql.util.PSQLException: Connection to compreface-postgres-db:6432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
2022-03-27 18:08:34.386  INFO 8 --- [           main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2022-03-27 18:08:34.489  INFO 8 --- [           main] ConditionEvaluationReportLoggingListener : 

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2022-03-27 18:08:34.505 ERROR 8 --- [           main] o.s.boot.SpringApplication               : Application run failed

org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in class path resource [org/springframework/boot/autoconfigure/liquibase/LiquibaseAutoConfiguration$LiquibaseConfiguration.class]: Invocation of init method failed; nested exception is liquibase.exception.DatabaseException: org.postgresql.util.PSQLException: Connection to compreface-postgres-db:6432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1794)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:594)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:516)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:324)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:322)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:311)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1109)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:869)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:551)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:143)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:758)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:750)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1237)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at com.exadel.frs.FrsApplication.main(FrsApplication.java:28)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:107)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:88)
Caused by: liquibase.exception.DatabaseException: org.postgresql.util.PSQLException: Connection to compreface-postgres-db:6432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:277)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1853)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1790)
	... 27 common frames omitted
Caused by: org.postgresql.util.PSQLException: Connection to compreface-postgres-db:6432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:303)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:51)
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:225)
	at org.postgresql.Driver.makeConnection(Driver.java:465)
	at org.postgresql.Driver.connect(Driver.java:264)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:358)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:477)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:560)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:112)
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:272)
	... 29 common frames omitted
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at java.base/java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.base/java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.base/java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.base/java.net.SocksSocketImpl.connect(Unknown Source)
	at java.base/java.net.Socket.connect(Unknown Source)
	at org.postgresql.core.PGStream.createSocket(PGStream.java:231)
	at org.postgresql.core.PGStream.<init>(PGStream.java:95)
	at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:98)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:213)
	... 41 common frames omitted
```

**Comment by pospielov** [comment]
Sorry for the long response
you have a different issue. `compreface-admin` container can't connect to the database
not sure why it says `compreface-postgres-db:6432`, the port should be 5432. Did you change it?

**Comment by Omar-Belghaouti** [comment]
@pospielov yes I did as you said in the above reply

> Looks like the container was stopped during the migration process. Did you try to update CompreFace? This is bad, I hope the DB is not corrupted. The easiest way is just to stop CompreFace, delete DB and start it again. But I suppose you don't want to delete the data. So you need to unblock DB manually. 0. Stop CompreFace
> 
> 1. You need to update your docker-compose.yml file. Add:
> 
> ```
> ports:
>       - "6432:5432"
> ```
> 
> to `compreface-postgres-db` container, like here: https://github.com/exadel-inc/CompreFace/blob/master/dev/docker-compose.yml
> 
> 2. Connect to DB:
>    user/pass: postgres/postgres
>    port: 6432
>    db: frs
> 3. Then find table `public.databasechangeloglock` and delete the row
> 4. Then you can start CompreFace again

**Comment by pospielov** [comment]
after point 3 "Then find table public.databasechangeloglock and delete the row" - it's good to revert all changes in `docker-compose.yml` and `.env`

**Comment by agusibrahim** [comment]
<img width="576" alt="Screen Shot 2022-07-09 at 08 56 22" src="https://user-images.githubusercontent.com/5948377/178087429-e7aa9ab9-cb93-4849-8d12-496cf8e2ed45.png">



i've facing same issue, but maybe different problem
here is the log
```
pymobile123@cloudshell:~$ docker logs compreface-ui
/docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
/docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
/docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
10-listen-on-ipv6-by-default.sh: info: /etc/nginx/conf.d/default.conf is not a file or does not exist
/docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
/docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
/docker-entrypoint.sh: Configuration complete; ready for start up
/docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
/docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
/docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
10-listen-on-ipv6-by-default.sh: info: /etc/nginx/conf.d/default.conf is not a file or does not exist
/docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
/docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
/docker-entrypoint.sh: Configuration complete; ready for start up
172.20.0.1 - - [09/Jul/2022:01:45:45 +0000] "ET /login HTTP/1.1" 405 552 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
2022/07/09 01:45:52 [error] 22#22: *1 connect() failed (113: No route to host) while connecting to upstream, client: 172.20.0.1, server: ui, request: "GET /admin/user/me HTTP/1.1", upstream: "http://172.20.0.5:8080/user/me", host: "127.0.0.1:8000", referrer: "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/"
172.20.0.1 - - [09/Jul/2022:01:45:52 +0000] "GET /admin/user/me HTTP/1.1" 502 552 "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
172.20.0.1 - - [09/Jul/2022:01:45:53 +0000] "GET / HTTP/1.1" 304 0 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
2022/07/09 01:45:55 [error] 22#22: *1 connect() failed (113: No route to host) while connecting to upstream, client: 172.20.0.1, server: ui, request: "GET /admin/user/me HTTP/1.1", upstream: "http://172.20.0.5:8080/user/me", host: "127.0.0.1:8000", referrer: "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/"
172.20.0.1 - - [09/Jul/2022:01:45:55 +0000] "GET /admin/user/me HTTP/1.1" 502 552 "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
172.20.0.1 - - [09/Jul/2022:01:45:58 +0000] "GET / HTTP/1.1" 304 0 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
2022/07/09 01:45:58 [error] 22#22: *1 connect() failed (113: No route to host) while connecting to upstream, client: 172.20.0.1, server: ui, request: "GET /admin/user/me HTTP/1.1", upstream: "http://172.20.0.5:8080/user/me", host: "127.0.0.1:8000", referrer: "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/"
172.20.0.1 - - [09/Jul/2022:01:45:58 +0000] "GET /admin/user/me HTTP/1.1" 502 552 "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
172.20.0.1 - - [09/Jul/2022:01:46:02 +0000] "GET /login HTTP/1.1" 200 1076 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
172.20.0.1 - - [09/Jul/2022:01:46:04 +0000] "GET /api/v1/consistence/status HTTP/1.1" 200 105 "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/login" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
2022/07/09 01:46:07 [error] 22#22: *1 connect() failed (113: No route to host) while connecting to upstream, client: 172.20.0.1, server: ui, request: "GET /admin/user/demo/model HTTP/1.1", upstream: "http://172.20.0.5:8080/user/demo/model", host: "127.0.0.1:8000", referrer: "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/login"
172.20.0.1 - - [09/Jul/2022:01:46:07 +0000] "GET /admin/user/demo/model HTTP/1.1" 502 552 "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/login" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
172.20.0.1 - - [09/Jul/2022:01:46:12 +0000] "GET /login HTTP/1.1" 304 0 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
172.20.0.1 - - [09/Jul/2022:01:46:12 +0000] "GET /api/v1/consistence/status HTTP/1.1" 200 105 "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/login" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
2022/07/09 01:46:16 [error] 22#22: *1 connect() failed (113: No route to host) while connecting to upstream, client: 172.20.0.1, server: ui, request: "GET /admin/user/demo/model HTTP/1.1", upstream: "http://172.20.0.5:8080/user/demo/model", host: "127.0.0.1:8000", referrer: "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/login"
172.20.0.1 - - [09/Jul/2022:01:46:16 +0000] "GET /admin/user/demo/model HTTP/1.1" 502 552 "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/login" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
172.20.0.1 - - [09/Jul/2022:01:49:19 +0000] "GET /demo HTTP/1.1" 200 1076 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
172.20.0.1 - - [09/Jul/2022:01:49:20 +0000] "GET /api/v1/consistence/status HTTP/1.1" 200 105 "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/demo" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
2022/07/09 01:49:23 [error] 22#22: *9 connect() failed (113: No route to host) while connecting to upstream, client: 172.20.0.1, server: ui, request: "GET /admin/user/demo/model HTTP/1.1", upstream: "http://172.20.0.5:8080/user/demo/model", host: "127.0.0.1:8000", referrer: "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/demo"
172.20.0.1 - - [09/Jul/2022:01:49:23 +0000] "GET /admin/user/demo/model HTTP/1.1" 502 552 "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/demo" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
2022/07/09 01:49:26 [error] 22#22: *9 connect() failed (113: No route to host) while connecting to upstream, client: 172.20.0.1, server: ui, request: "GET /admin/user/me HTTP/1.1", upstream: "http://172.20.0.5:8080/user/me", host: "127.0.0.1:8000", referrer: "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/"
172.20.0.1 - - [09/Jul/2022:01:49:26 +0000] "GET /admin/user/me HTTP/1.1" 502 552 "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
172.20.0.1 - - [09/Jul/2022:01:49:31 +0000] "GET /api/v1/consistence/status HTTP/1.1" 200 105 "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/demo" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
2022/07/09 01:49:34 [error] 22#22: *9 connect() failed (113: No route to host) while connecting to upstream, client: 172.20.0.1, server: ui, request: "GET /admin/user/demo/model HTTP/1.1", upstream: "http://172.20.0.5:8080/user/demo/model", host: "127.0.0.1:8000", referrer: "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/demo"
172.20.0.1 - - [09/Jul/2022:01:49:34 +0000] "GET /admin/user/demo/model HTTP/1.1" 502 552 "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/demo" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
2022/07/09 01:49:37 [error] 22#22: *9 connect() failed (113: No route to host) while connecting to upstream, client: 172.20.0.1, server: ui, request: "GET /admin/user/me HTTP/1.1", upstream: "http://172.20.0.5:8080/user/me", host: "127.0.0.1:8000", referrer: "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/"
172.20.0.1 - - [09/Jul/2022:01:49:37 +0000] "GET /admin/user/me HTTP/1.1" 502 552 "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
172.20.0.1 - - [09/Jul/2022:01:50:29 +0000] "GET / HTTP/1.1" 304 0 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
2022/07/09 01:50:33 [error] 22#22: *9 connect() failed (113: No route to host) while connecting to upstream, client: 172.20.0.1, server: ui, request: "GET /admin/user/me HTTP/1.1", upstream: "http://172.20.0.5:8080/user/me", host: "127.0.0.1:8000", referrer: "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/"
172.20.0.1 - - [09/Jul/2022:01:50:33 +0000] "GET /admin/user/me HTTP/1.1" 502 552 "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
172.20.0.1 - - [09/Jul/2022:01:55:42 +0000] "GET /api/v1/consistence/status HTTP/1.1" 200 105 "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/login" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
2022/07/09 01:55:45 [error] 22#22: *17 connect() failed (113: No route to host) while connecting to upstream, client: 172.20.0.1, server: ui, request: "GET /admin/user/demo/model HTTP/1.1", upstream: "http://172.20.0.5:8080/user/demo/model", host: "127.0.0.1:8000", referrer: "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/login"
172.20.0.1 - - [09/Jul/2022:01:55:45 +0000] "GET /admin/user/demo/model HTTP/1.1" 502 552 "https://8000-64409127-388d-4786-a9f0-db843a3bcf7e.cs-asia-southeast1-ajrg.cloudshell.dev/login" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
```

```
pymobile123@cloudshell:~$ docker logs compreface-admin
Listening for transport dt_socket at address: 5005
2022-07-09 01:39:58.320  INFO 8 --- [kground-preinit] o.h.validator.internal.util.Version      : HV000001: Hibernate Validator 6.1.5.Final

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::        (v2.3.4.RELEASE)

2022-07-09 01:40:00.328  INFO 8 --- [           main] com.exadel.frs.FrsApplication            : The following profiles are active: dev
2022-07-09 01:40:05.257  INFO 8 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFERRED mode.
2022-07-09 01:40:05.898  INFO 8 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 575ms. Found 10 JPA repository interfaces.
2022-07-09 01:40:06.958  INFO 8 --- [           main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=e7f41a96-3f70-3efd-b7dc-40aa4699bf29
2022-07-09 01:40:07.266  INFO 8 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'com.exadel.frs.commonservice.system.feign.ApperyStatisticsClient' of type [org.springframework.cloud.openfeign.FeignClientFactoryBean] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2022-07-09 01:40:09.900  INFO 8 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2022-07-09 01:40:09.949  INFO 8 --- [           main] o.a.coyote.http11.Http11NioProtocol      : Initializing ProtocolHandler ["http-nio-8080"]
2022-07-09 01:40:09.950  INFO 8 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2022-07-09 01:40:09.950  INFO 8 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.38]
2022-07-09 01:40:10.299  INFO 8 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2022-07-09 01:40:10.300  INFO 8 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 9921 ms
2022-07-09 01:40:11.255  INFO 8 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2022-07-09 01:40:11.712  INFO 8 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2022-07-09 01:40:13.286  INFO 8 --- [           main] liquibase.lockservice                    : Successfully acquired change log lock
2022-07-09 01:40:15.171  INFO 8 --- [           main] liquibase.changelog                      : Reading from public.databasechangelog
2022-07-09 01:40:15.794  INFO 8 --- [           main] liquibase.lockservice                    : Successfully released change log lock
2022-07-09 01:40:15.803  INFO 8 --- [           main] liquibase.lockservice                    : Successfully acquired change log lock
Skipping auto-registration
2022-07-09 01:40:15.804  WARN 8 --- [           main] liquibase.hub                            : Skipping auto-registration
2022-07-09 01:40:15.846  INFO 8 --- [           main] liquibase.lockservice                    : Successfully released change log lock
2022-07-09 01:40:16.248  INFO 8 --- [           main] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'applicationTaskExecutor'
2022-07-09 01:40:16.315  INFO 8 --- [           main] o.s.s.c.ThreadPoolTaskScheduler          : Initializing ExecutorService 'taskScheduler'
2022-07-09 01:40:16.833  INFO 8 --- [         task-1] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2022-07-09 01:40:17.369  INFO 8 --- [         task-1] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 5.4.21.Final
2022-07-09 01:40:18.090  INFO 8 --- [         task-1] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
2022-07-09 01:40:19.277  INFO 8 --- [         task-1] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.PostgreSQL10Dialect
2022-07-09 01:40:20.131  INFO 8 --- [           main] DeferredRepositoryInitializationListener : Triggering deferred initialization of Spring Data repositories…
2022-07-09 01:40:20.482  WARN 8 --- [         task-1] Hibernate Types                          : You should use Hypersistence Optimizer to speed up your Hibernate application!
2022-07-09 01:40:20.483  WARN 8 --- [         task-1] Hibernate Types                          : For more details, go to https://vladmihalcea.com/hypersistence-optimizer/
2022-07-09 01:40:20.484  INFO 8 --- [         task-1] Hibernate Types                          :
 _    _                           _     _
| |  | |                         (_)   | |
| |__| |_   _ _ __   ___ _ __ ___ _ ___| |_ ___ _ __   ___ ___
|  __  | | | | '_ \ / _ \ '__/ __| / __| __/ _ \ '_ \ / __/ _ \
| |  | | |_| | |_) |  __/ |  \__ \ \__ \ ||  __/ | | | (_|  __/
|_|  |_|\__, | .__/ \___|_|  |___/_|___/\__\___|_| |_|\___\___|
         __/ | |
        |___/|_|

           ____        _   _           _
          / __ \      | | (_)         (_)
         | |  | |_ __ | |_ _ _ __ ___  _ _______ _ __
         | |  | | '_ \| __| | '_ ` _ \| |_  / _ \ '__|
         | |__| | |_) | |_| | | | | | | |/ /  __/ |
          \____/| .__/ \__|_|_| |_| |_|_/___\___|_|
                | |
                |_|

2022-07-09 01:40:20.484  INFO 8 --- [         task-1] Hibernate Types                          : Check out the README page for more info about the Hypersistence Optimizer banner https://github.com/vladmihalcea/hibernate-types#how-to-remove-the-hypersistence-optimizer-banner-from-the-log
2022-07-09 01:40:22.792  INFO 8 --- [         task-1] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2022-07-09 01:40:22.824  INFO 8 --- [         task-1] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2022-07-09 01:40:25.456  INFO 8 --- [           main] DeferredRepositoryInitializationListener : Spring Data repositories initialized!
2022-07-09 01:40:26.641  INFO 8 --- [           main] o.s.b.a.e.web.EndpointLinksResolver      : Exposing 2 endpoint(s) beneath base path '/actuator'
2022-07-09 01:40:26.901  INFO 8 --- [           main] pertySourcedRequestMappingHandlerMapping : Mapped URL path [/v2/api-docs] onto method [springfox.documentation.swagger2.web.Swagger2Controller#getDocumentation(String, HttpServletRequest)]
2022-07-09 01:40:28.266  INFO 8 --- [           main] o.s.s.web.DefaultSecurityFilterChain     : Creating filter chain: Ant [pattern='/v2/api-docs'], []
2022-07-09 01:40:28.269  INFO 8 --- [           main] o.s.s.web.DefaultSecurityFilterChain     : Creating filter chain: Ant [pattern='/configuration/ui'], []
2022-07-09 01:40:28.269  INFO 8 --- [           main] o.s.s.web.DefaultSecurityFilterChain     : Creating filter chain: Ant [pattern='/swagger-resources/**'], []
2022-07-09 01:40:28.269  INFO 8 --- [           main] o.s.s.web.DefaultSecurityFilterChain     : Creating filter chain: Ant [pattern='/configuration/**'], []
2022-07-09 01:40:28.269  INFO 8 --- [           main] o.s.s.web.DefaultSecurityFilterChain     : Creating filter chain: Ant [pattern='/swagger-ui**'], []
2022-07-09 01:40:28.269  INFO 8 --- [           main] o.s.s.web.DefaultSecurityFilterChain     : Creating filter chain: Ant [pattern='/webjars/**'], []
2022-07-09 01:40:28.269  INFO 8 --- [           main] o.s.s.web.DefaultSecurityFilterChain     : Creating filter chain: Ant [pattern='/lms/**'], []
2022-07-09 01:40:28.361  INFO 8 --- [           main] o.s.s.web.DefaultSecurityFilterChain     : Creating filter chain: OrRequestMatcher [requestMatchers=[Ant [pattern='/oauth/token'], Ant [pattern='/oauth/token_key'], Ant [pattern='/oauth/check_token']]], [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@53d808ea, org.springframework.security.web.context.SecurityContextPersistenceFilter@3e839aa3, org.springframework.security.web.header.HeaderWriterFilter@37e64e37, org.springframework.security.web.authentication.logout.LogoutFilter@4f563f96, org.springframework.security.web.authentication.www.BasicAuthenticationFilter@5ca86715, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@751bb780, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@416d90f0, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@44864ebe, org.springframework.security.web.session.SessionManagementFilter@d08f85a, org.springframework.security.web.access.ExceptionTranslationFilter@511de4dd, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@bd28c86]
2022-07-09 01:40:28.390  INFO 8 --- [           main] o.s.s.web.DefaultSecurityFilterChain     : Creating filter chain: org.springframework.security.oauth2.config.annotation.web.configuration.ResourceServerConfiguration$NotOAuthRequestMatcher@15f11bfb, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@16a499d1, org.springframework.security.web.context.SecurityContextPersistenceFilter@2e4339c1, org.springframework.security.web.header.HeaderWriterFilter@1c2fb9e8, org.springframework.security.web.authentication.logout.LogoutFilter@552088cc, org.springframework.security.oauth2.provider.authentication.OAuth2AuthenticationProcessingFilter@1c2b65cc, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@62dc1203, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@35d0749, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@2bf4fa1, org.springframework.security.web.session.SessionManagementFilter@ec3944, org.springframework.security.web.access.ExceptionTranslationFilter@ef85567, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@520ee6b3]
2022-07-09 01:40:28.402  INFO 8 --- [           main] o.s.s.web.DefaultSecurityFilterChain     : Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@570299e3, org.springframework.security.web.context.SecurityContextPersistenceFilter@71df5f30, org.springframework.security.web.header.HeaderWriterFilter@5ebe903a, org.springframework.security.web.authentication.logout.LogoutFilter@539dd2d0, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@4895e8f6, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@62808e9e, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@bff764c, org.springframework.security.web.session.SessionManagementFilter@7c88d04f, org.springframework.security.web.access.ExceptionTranslationFilter@2e62e227, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@5ad50b02]
2022-07-09 01:40:28.868  INFO 8 --- [           main] org.quartz.impl.StdSchedulerFactory      : Using default implementation for ThreadExecutor
2022-07-09 01:40:28.953  INFO 8 --- [           main] org.quartz.core.SchedulerSignalerImpl    : Initialized Scheduler Signaller of type: class org.quartz.core.SchedulerSignalerImpl
2022-07-09 01:40:28.953  INFO 8 --- [           main] org.quartz.core.QuartzScheduler          : Quartz Scheduler v.2.3.2 created.
2022-07-09 01:40:28.958  INFO 8 --- [           main] o.s.s.quartz.LocalDataSourceJobStore     : Using db table-based data access locking (synchronization).
2022-07-09 01:40:28.962  INFO 8 --- [           main] o.s.s.quartz.LocalDataSourceJobStore     : JobStoreCMT initialized.
2022-07-09 01:40:28.978  INFO 8 --- [           main] org.quartz.core.QuartzScheduler          : Scheduler meta-data: Quartz Scheduler (v2.3.2) 'statistics scheduler' with instanceId 'NON_CLUSTERED'
  Scheduler class: 'org.quartz.core.QuartzScheduler' - running locally.
  NOT STARTED.
  Currently in standby mode.
  Number of jobs executed: 0
  Using thread pool 'org.quartz.simpl.SimpleThreadPool' - with 1 threads.
  Using job-store 'org.springframework.scheduling.quartz.LocalDataSourceJobStore' - which supports persistence. and is not clustered.

2022-07-09 01:40:28.978  INFO 8 --- [           main] org.quartz.impl.StdSchedulerFactory      : Quartz scheduler 'statistics scheduler' initialized from an externally provided properties instance.
2022-07-09 01:40:28.978  INFO 8 --- [           main] org.quartz.impl.StdSchedulerFactory      : Quartz scheduler version: 2.3.2
2022-07-09 01:40:28.978  INFO 8 --- [           main] org.quartz.core.QuartzScheduler          : JobFactory set to: org.springframework.scheduling.quartz.SpringBeanJobFactory@490d9c41
2022-07-09 01:40:29.150  INFO 8 --- [           main] o.s.s.quartz.LocalDataSourceJobStore     : Freed 0 triggers from 'acquired' / 'blocked' state.
2022-07-09 01:40:29.155  INFO 8 --- [           main] o.s.s.quartz.LocalDataSourceJobStore     : Recovering 0 jobs that were in-progress at the time of the last shut-down.
2022-07-09 01:40:29.155  INFO 8 --- [           main] o.s.s.quartz.LocalDataSourceJobStore     : Recovery complete.
2022-07-09 01:40:29.157  INFO 8 --- [           main] o.s.s.quartz.LocalDataSourceJobStore     : Removed 0 'complete' triggers.
2022-07-09 01:40:29.157  INFO 8 --- [           main] o.s.s.quartz.LocalDataSourceJobStore     : Removed 0 stale fired job entries.
2022-07-09 01:40:29.161  INFO 8 --- [           main] org.quartz.core.QuartzScheduler          : Scheduler statistics scheduler_$_NON_CLUSTERED started.
2022-07-09 01:40:29.278  WARN 8 --- [           main] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2022-07-09 01:40:29.286  INFO 8 --- [           main] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2022-07-09 01:40:29.343  WARN 8 --- [           main] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2022-07-09 01:40:29.363  INFO 8 --- [           main] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2022-07-09 01:40:30.722  INFO 8 --- [           main] o.a.coyote.http11.Http11NioProtocol      : Starting ProtocolHandler ["http-nio-8080"]
2022-07-09 01:40:30.850  INFO 8 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8080 (http) with context path ''
2022-07-09 01:40:30.863  INFO 8 --- [           main] d.s.w.p.DocumentationPluginsBootstrapper : Context refreshed
2022-07-09 01:40:31.199  INFO 8 --- [           main] d.s.w.p.DocumentationPluginsBootstrapper : Found 1 custom documentation plugin(s)
2022-07-09 01:40:31.478  INFO 8 --- [           main] s.d.s.w.s.ApiListingReferenceScanner     : Scanning for api listing references
2022-07-09 01:40:32.139  INFO 8 --- [           main] .d.s.w.r.o.CachingOperationNameGenerator : Generating unique operation named: regenerateApiKeyUsingPUT_1
2022-07-09 01:40:32.377  INFO 8 --- [           main] DeferredRepositoryInitializationListener : Triggering deferred initialization of Spring Data repositories…
2022-07-09 01:40:32.378  INFO 8 --- [           main] DeferredRepositoryInitializationListener : Spring Data repositories initialized!
2022-07-09 01:40:32.459  INFO 8 --- [           main] com.exadel.frs.FrsApplication            : Started FrsApplication in 36.596 seconds (JVM running for 40.019)
```

**Comment by agusibrahim** [comment]
> 

i switching to other cloud platform instead of google cloud shell, it's working perfectly without any modification. thanx
